lib/components-schema-v1_7_x.ts

```diff
@@ -35,6 +35,48 @@ function labelProperty(description: string): { oneOf: JSONSchema7Definition[] }
     };
 }
 
+const inlineComponentPropertyDefinitionOrReferenceList: { oneOf: JSONSchema7Definition[] } = {
+    oneOf: [
+        {
+            type: 'string',
+        },
+        {
+            type: 'object',
+            additionalProperties: false,
+            required: ['name', 'directiveKey'],
+            properties: {
+                name: {
+                    type: 'string',
+                    description: 'component property identifier',
```

Comment:
Small note: this description starts without capital, the other start with one. In the rest of the most descriptions start with a capital, though not all of them.